### PR TITLE
Minimal Signal Changes Part 1 [win64amd]

### PR DESCRIPTION
**Pull request by babsingh**
- **Add a new port library flag for SIGBREAK**

    Currently, port library flag for SIGQUIT is used for SIGBREAK. SIGQUIT and
    SIGBREAK are different signals. SIGBREAK only exists on Windows. SIGQUIT
    doesn't exist on Windows. A different port library flag must be used for
    SIGBREAK.

- **Add signalMap for win64amd**

    signalMap includes the list of signals supported by a platform.

    OMRPORT_SIG_FLAG_SIGALLASYNC is a bit-mask of all the asynchronous
    signals that OMR signal library supports on all platforms.

    OMRPORT_SIG_FLAG_SIGALLSYNC is a bit-mask of all the synchronous signals
    that OMR signal library supports on all the platforms.

    signalMap has been added for win64amd. It specifies the signals which
    are supported on win64amd.

- **Add implementation of omrsig_map_os_signal_to_portlib_signal (win64amd)**

    Added a static method, mapOSSignalToPortLib, which is called from
    omrsig_map_portlib_signal_to_os_signal. mapOSSignalToPortLib will be
    used from other places in the future.

- **Add implementation of omrsig_map_portlib_signal_to_os_signal (win64amd)**

    Added a static method, mapPortLibSignalToOSSignal, which is called from
    omrsig_map_portlib_signal_to_os_signal. mapPortLibSignalToOSSignal will
    be used from other places in the future.

    Signed-off-by: Babneet Singh <sbabneet@ca.ibm.com>

### CODE REVIEW

**Comment by babsingh** [comment]
fyi - @pshipton

File: port/common/omrport.tdf [hunk 1]
```diff
@@ -1075,3 +1075,6 @@ TraceExit=Trc_PRT_signal_omrsig_is_master_signal_handler_exiting Group=signal Ov
 
 TraceException=Trc_PRT_readCgroupFile_fgets_failed Group=sysinfo Overhead=1 Level=3 NoEnv Template="readCgroupFile: fgets failed for %s with errno=%d"
 TraceException=Trc_PRT_isRunningInContainer_fgets_failed Group=sysinfo Overhead=1 Level=3 NoEnv Template="isRunningInContainer: unexpected format of file %s with errno=%d"
+
+TraceEvent=Trc_PRT_signal_mapPortLibSignalToOSSignal_ERROR_unknown_signal Group=signal Overhead=1 Level=3 NoEnv Template="omrsignal: mapPortLibSignalToOSSignal: ERROR, unknown signal, portLibSignal=0x%X"
```
Comment (pshipton) [inline port/common/omrport.tdf]: Should be a TraceException. As should Trc_PRT_signal_mapPortLibSignalToUnix_ERROR_unknown_signal.

Comment (babsingh) [reply port/common/omrport.tdf]: updated definition of `Trc_PRT_signal_mapPortLibSignalToOSSignal_ERROR_unknown_signal`. will set `Trc_PRT_signal_mapPortLibSignalToUnix_ERROR_unknown_signal` to obsolete and instead, use `Trc_PRT_signal_mapPortLibSignalToUnix_ERROR_unknown_signal`.

**Comment by charliegracie** [comment]
@genie-omr build all

File: port/common/omrport.tdf (hunk 1 above)
Comment (babsingh) [reply port/common/omrport.tdf]: i will add a commit to replace `Trc_PRT_signal_mapPortLibSignalToUnix_ERROR_unknown_signal` with `Trc_PRT_signal_mapPortLibSignalToOSSignal_ERROR_unknown_signal` in the next set of changes (**Part 2**).

**Comment by charliegracie** [comment]
@pshipton are you ok with these changes?

**Comment by pshipton** [comment]
lgtm

**Pull request merged**